ql/src/test/results/clientpositive/llap/groupby3_map_multi_distinct.q.out

```diff
@@ -69,12 +69,12 @@ STAGE PLANS:
                       minReductionHashAggr: 0.4
                       mode: hash
                       outputColumnNames: _col0, _col1, _col2, _col3, _col4, _col5, _col6, _col7, _col8, _col9, _col10, _col11
-                      Statistics: Num rows: 500 Data size: 263000 Basic stats: COMPLETE Column stats: COMPLETE
+                      Statistics: Num rows: 307 Data size: 161482 Basic stats: COMPLETE Column stats: COMPLETE
```

Comment:
I wonder why this size is reduced

Reply:
@okumin the reduction in size here is due to the map side aggregation (which is set to true as part of this qtest) which happens when tez container size was increased to 256. This aggregation did not run due to insufficient memory earlier. I have added the detailed description related to this change in the JIRA association with the commit here: [HIVE-28396](https://issues.apache.org/jira/browse/HIVE-28396) . Please take a look.